src/js/components/replenishment/CreateReplenishment.jsx

```diff
@@ -26,6 +27,25 @@ const FIELD = {
     type: ArrayField,
     arrowsNavigation: true,
     fields: {
+      checked: {
+        type: CheckboxField,
+        label: 'react.stockMovement.selectAll.label',
+        defaultMessage: 'Select All',
+        flexWidth: 4,
+        getDynamicAttr: ({
+          rowIndex, isAllProductsSelected, selectAllCode, updateSelectedItems,
+        }) => ({
+          headerHtml: () => (
+            <input
+              type="checkbox"
+              className="mt-1"
+              checked={isAllProductsSelected}
```

Comment:
Should be `allRowsSelected`, since there might be multiple occurrences of the same product and the naming here might be misleading.